docs/deployment-methods.md

```diff
@@ -41,4 +42,4 @@ Webhooks can be added here:
 - Project > Settings > Webhooks > Add Webhook > Title: Captain Server, URL: Captain Webhook from your apps page.
 
 #### GitLab and Others
-Webhooks can be added in a similar fashion. As long as the webhook fires a POST request, CapRover is able to pick it up and starts a build from the latest commit on the specified branch.
\ No newline at end of file
+Webhooks can be added in a similar fashion. As long as the webhook fires a POST request, CapRover is able to pick it up and starts a build from the latest commit on the specified branch.
```

Comment:
Dont know why it highlights this. Have nothing changed here

Reply:
It happened for me as well - presumably when you edit through GitHub directly, it changes the EOF format